release-notes/PTP/PTP_1.1.3_CU2.md

```diff
@@ -0,0 +1,79 @@
+---
+uid: PTP_1.1.3_CU2
+---
+
+# PTP 1.1.3 CU2
+
+## Enhancements
+
+#### New parameters added to standard PTP mediation layer [ID_34993]
+
+Parameters for the following connectors that were still missing in the mediation layer have now been added:
```

Comment:
Cisco Nexus was not changed with this RN. Instead, it only adding support for the PTP Mode of the Arista Manager. 
Tektronix is covered in the other RNs